### PR TITLE
ENG-637 New endpoint to download trashed resources

### PR DESCRIPTION
**Pull request by BrenoQVDS**

### CODE REVIEW

File: plugins/entando-plugin-jpversioning/src/test/java/org/entando/entando/jpversioning/web/resource/ResourceVersioningControllerIntegrationTest.java [hunk 1]
```diff
@@ -635,26 +771,12 @@ private ResultActions performCreateResource(UserDetails user, String type, Strin
...
-                "a;lsdka;lsdka;lsdka;lsdk;alskd;laskd;aslkd;alsdk;alskda;lskldaskl;sdjodpasu0i9728938701o7i186r890347974209817409823740bgbdf98dw787012378b1789b13281328701b39871029371x" +
-                "a;lsdka;lsdka;lsdka;lsdk;alskd;laskd;aslkd;alsdk;alskda;lskldaskl;sdjodpasu0i9728938701o7i186r890347974209817409823740bgbdf98dw787012378b1789b13281328701b39871029371x";
 
         MockMultipartFile file;
         if ("image".equals(type)) {
-            file = new MockMultipartFile("file", "image_test.jpeg", mimeType, contents.getBytes());
+            file = new MockMultipartFile("file", "image_test.jpeg", mimeType, CONTENT.getBytes());
         } else {
-            file = new MockMultipartFile("file", "file_test.jpeg", mimeType, contents.getBytes());
+            file = new MockMultipartFile("file", "file_test.jpeg", mimeType, CONTENT.getBytes());
```
Comment (alepintus) [inline plugins/entando-plugin-jpversioning/src/test/java/org/entando/entando/jpversioning/web/resource/ResourceVersioningControllerIntegrationTest.java:779]: Hey @BrenoQVDS ,
is this file extension correct?

Comment (BrenoQVDS) [reply plugins/entando-plugin-jpversioning/src/test/java/org/entando/entando/jpversioning/web/resource/ResourceVersioningControllerIntegrationTest.java:779]: Hi @alepintus,
Yeah, this is just the filename for testing, its extension doesn't really matter, this is the same code we have in CMS, I just copied to Versioning. There are no checks related to that, the mimeType is what matter.

**Comment by entando-service** [comment]
All is Well

**Comment by alepintus** [comment]
Looks good

**Pull request merged**